docs/self-sufficient-bundle.md

```diff
@@ -23,7 +23,7 @@ related to their `*.service` counterparts and starts the service when the paths
 
 > [!NOTE]
 > "Marker env variable" is set using an env file, if the required env variable is not set then unit is skipped
-> some units are run only when CRC_CLOUD=1 is set, these are only needed when using the bundles with crc-cloud
+> some units are run only when CRC_NG=1 is set, these are only needed when using the bundles with crc-cloud
```

Comment:
The name `CRC_CLOUD` feels better to me, it’s more descriptive than `CRC_NG`, and is relatively accurate.
`CRC_STANDALONE_BUNDLE`, `CRC_STANDALONE_EXECUTION`, `CRC_NO_CRC` (not really serious about this one) may work too.

Reply:
`CRC_CLOUD` feels like this variable depend on running the crc on cloud env which is not the case so to avoid confusion I thought having `NG` => next gen would be much better. @crc-org/crc-team wdyt?

Reply:
Updated the readme about self sufficient bundle and add context for CRC_NG reference, hope that will make it bit easier to consume.